docker/usr/bin/entrypoint

```diff
@@ -22,6 +22,13 @@ for FOLDER in /data/gitea/conf /data/gitea/log /data/git /data/ssh; do
     mkdir -p ${FOLDER}
 done
 
+if [ -f /data/gitea/conf/app.ini ]; then
+    echo "Found app.ini config file, migrating database"
+    chmod 644 /data/gitea/conf/app.ini
+    chown -R 1000:1000 /data/git /data/gitea
+    su - git -c gitea migrate -c /data/gitea/conf/app.ini
+fi
```

Comment:
`${USER_UID}`, `${USER_GID}`, `$UID` and `$GID` are undefined, i'll user $USER:$USER is that ok ?

Reply:
The original entry point script just checks for the variables and if they're there does something. That's why they're undefined.

You may have to adjust it to define the variables if they're not there or empty.

```bash
if [ -z "${USER_GID}" ]; then
  USER_GID="`id -g ${USER}`"
fi
```

And similarly for UID. Otherwise I'd be suspicious that the chown could choose the wrong UID and GID in cases were they were set explicitly - but I'd happily defer to more expert opinion on this.

Reply:
Ah yes. You are right @pciavald, I missed that they were undefined. I think that @zeripath's approach of setting a default for the values is a good one.